### ISSUE TITLE
settings.CACHE_TIMEOUT Unknown Variable.

### ISSUE THREAD

**Issue by arky** [translate/pootle#3183]
Got this error while testing today's master. 

2014-02-24T18:08:54+0700 app.0: Traceback (most recent call last):
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 255, in **call**
2014-02-24T18:08:54+0700 app.0:     response = self.get_response(request)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/base.py", line 178, in get_response
2014-02-24T18:08:54+0700 app.0:     response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/base.py", line 224, in handle_uncaught_exception
2014-02-24T18:08:54+0700 app.0:     return callback(request, *_param_dict)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/utils/decorators.py", line 91, in _wrapped_view
2014-02-24T18:08:54+0700 app.0:     response = view_func(request, *args, *_kwargs)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/views/defaults.py", line 41, in server_error
2014-02-24T18:08:54+0700 app.0:     return http.HttpResponseServerError(template.render(Context({})))
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 140, in render
2014-02-24T18:08:54+0700 app.0:     return self._render(context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
2014-02-24T18:08:54+0700 app.0:     return self.nodelist.render(context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 830, in render
2014-02-24T18:08:54+0700 app.0:     bit = self.render_node(node, context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 844, in render_node
2014-02-24T18:08:54+0700 app.0:     return node.render(context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/loader_tags.py", line 124, in render
2014-02-24T18:08:54+0700 app.0:     return compiled_parent._render(context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/loader_tags.py", line 63, in render
2014-02-24T18:08:54+0700 app.0:     result = block.nodelist.render(context)
2014-02-24T18:08:54+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/templatetags/cache.py", line 23, in render
2014-02-24T18:08:54+0700 app.0:     raise TemplateSyntaxError('"cache" tag got an unknown variable: %r' % self.expire_time_var.var)
2014-02-24T18:08:54+0700 app.0: django.template.base.TemplateSyntaxError: "cache" tag got an unknown variable: u'settings.CACHE_TIMEOUT'
2014-02-24T18:08:54+0700 app.0: [pid: 15834|app: 0|req: 5/5] 10.22.94.30 () {22 vars in 260 bytes} [Mon Feb 24 11:08:54 2014] GET / => generated 0 bytes in 386 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
2014-02-24T18:09:26+0700 app.0: Traceback (most recent call last):
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 255, in __call__
2014-02-24T18:09:26+0700 app.0:     response = self.get_response(request)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/base.py", line 178, in get_response
2014-02-24T18:09:26+0700 app.0:     response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/core/handlers/base.py", line 224, in handle_uncaught_exception
2014-02-24T18:09:26+0700 app.0:     return callback(request, *_param_dict)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/utils/decorators.py", line 91, in _wrapped_view
2014-02-24T18:09:26+0700 app.0:     response = view_func(request, *args, *_kwargs)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/views/defaults.py", line 41, in server_error
2014-02-24T18:09:26+0700 app.0:     return http.HttpResponseServerError(template.render(Context({})))
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 140, in render
2014-02-24T18:09:26+0700 app.0:     return self._render(context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
2014-02-24T18:09:26+0700 app.0:     return self.nodelist.render(context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 830, in render
2014-02-24T18:09:26+0700 app.0:     bit = self.render_node(node, context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/base.py", line 844, in render_node
2014-02-24T18:09:26+0700 app.0:     return node.render(context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/loader_tags.py", line 124, in render
2014-02-24T18:09:26+0700 app.0:     return compiled_parent._render(context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/template/loader_tags.py", line 63, in render
2014-02-24T18:09:26+0700 app.0:     result = block.nodelist.render(context)
2014-02-24T18:09:26+0700 app.0:   File "/app/python/lib/python2.7/site-packages/django/templatetags/cache.py", line 23, in render
2014-02-24T18:09:26+0700 app.0:     raise TemplateSyntaxError('"cache" tag got an unknown variable: %r' % self.expire_time_var.var)

**Comment by avesus** [comment]
Doesn't fixed on 2.5.1.1 on CentOS 6.

**Comment by glensc** [comment]
indeed, isn't fixed with 2.5.1.3 either. using default config produced by `pootle init`, and my `ALLOWED_HOSTS = ['*']`

**Comment by glensc** [comment]
the real fix was to enable cache which indirectly was suggested from
https://github.com/translate/pootle/issues/3183#issuecomment-50272041

as i am installing 2.5 only to upgrade from 2.1 to 2.7, so i just configured dummy cache:

```
CACHES = {
    'default': {
        'BACKEND': 'django.core.cache.backends.dummy.DummyCache'
    } 
}
```
